doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20-rc1.rst

```diff
@@ -0,0 +1,432 @@
+.. _ncs_release_notes_290-nrf54h20-rc1:
+
+|NCS| v2.9.0-nRF54H20-rc1 Release Notes
+#######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+The |NCS| v2.9.0-nrf54h20-rc1 is a nRF54H20-exclusive release tag, only supporting products based on the nRF54H20 SoC.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+Added the following features as supported:
+
+* nRF54H20 Series:
+
+  * Triggering firmware recovery by pressing a hardware button or calling the respective API.
+  * Dynamic Global Domain Frequency Scaling (DGFS), optimizing USB suspend power consumption.
+  * Drivers adapted for the following peripherals:
+
+    * UARTE120
+    * PWM120
+    * SPIM120
+    * EXMIF
+
+  * UICR validation support.
+  * MSPI EXMIF support.
+
+* Power Management:
+
+  * System-On-All-Idle power improvements.
+  * Thread/Matter power efficiency improvements.
```

Comment:
This is not part of the tag release yet, would be part of 3.0.0. So for.now i would recommend to remove this line.